package.json

```diff
@@ -60,7 +58,7 @@
     "postcss-load-config": "^2.1.0",
     "postcss-preset-env": "^6.7.0",
     "postcss-reporter": "^6.0.1",
-    "react": "^15.3.2",
+    "react": "^16.13.1",
```

Comment:
Do we still need this? If not, can we remove and regenerate the lockfile?

Reply:
Sorry, I was going to address this, but looks like you merged. I'm sure we can fix this up in a future PR. 👍

Reply:
Yeah no worries. I'm planning to remove a bunch of other deps in https://github.com/sparkbox/bouncy-ball/issues/77.